yarn.lock

```diff
@@ -5720,9 +5720,9 @@ human-signals@^2.1.0:
   resolved "https://registry.yarnpkg.com/human-signals/-/human-signals-2.1.0.tgz#dc91fcba42e4d06e4abaed33b3e7a3c02f514ea0"
   integrity sha512-B4FFZ6q/T2jhhksgkbEW3HBvWIfDW85snkQgawt07S7J5QXTk6BkNV+0yAeZrM5QpMAdYlocGoljn0sJ/WQkFw==
 
-hydrogen-web@Automattic/hydrogen-web#chatrix-0.7.1:
+hydrogen-web@Automattic/hydrogen-web#e1f235bda075d06b1f475f4d5dfa3bb3bf07ad0b:
```

Comment:
@psrpinto Should we merge [the branch](https://github.com/Automattic/hydrogen-web/pull/19/) first, and then use merge commit's hash instead? That way its always a commit on master branch of our fork.

Reply:
I was planning to do so when preparing the release, if that's ok with you.

Reply:
I will create a tag and use that, `chatrix-0.9.0`.

Reply:
Any reason for doing so? Seems better to merge that branch first and then update the dependency in this PR, since that's where updation of hydrogen is relevant?

Reply:
No specific reason for doing so, I agree it's better to depend on the merge commit. I will do so here.

> Next version should be 0.8?

Correct, I mis-remembered.